src/Arcus.Security.Tests.Runtimes.AzureFunctions/OrderFunction.cs

```diff
@@ -29,13 +28,16 @@ public OrderFunction(ISecretProvider secretProvider, ILogger<OrderFunction> logg
             _secretProvider = secretProvider;
         }
 
-        [FunctionName("order")]
-        public async Task<IActionResult> Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest request,
-            ILogger log)
+        [Function("order")]
+        public async Task<HttpResponseData> Run(
```

Comment:
So, we need to always return HttpResponseData instead of IActionResult ?  What is the difference ?

Reply:
This is now an isolated Azure Function. There is a new feature that would allow us to still use ASP.NET Core functionality, but wanted to minimize the project here. `HttpRequestData`/`HttpResponseData` is the default approach for isolated HTTP trigged Azure Functions.